### ISSUE TITLE
How to best do stable releases?

### ISSUE THREAD

**Issue by jacqui** [news-catalyst/next-tinynewsdemo#1166]
Perhaps follow the GH process? https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository

What we've been doing in the days since launching the new version of the platform is:

* merging PRs into main
* locally, in terminal, merging main into the stable branch
* then pushing the latest stable branch to origin (github)
* which then tells vercel to do a production deploy

I'm thinking the part that needs fixing is the two commands run in localhost terminal

**Comment by jacqui** [comment]
Okay, I just did a trial run of the GH release process linked in this issue's description, and I don't think it's a good fit for our use case.

What we want is a process that will take the latest code in the `main` branch and merge it into the `stable` branch. As I listed above, this is only a few commands. When I used [git-flow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow), doing a "release" like that was pretty simple and followed a structured set of steps (`git flow release start 1.0.1`, write some release notes, `git flow release finish version.number.here`). We're basically following the git-flow approach with feature branches, a develop branch, and a main branch. A little confusingly, we've called the git-flow "develop" branch "main" and the git-flow "main" branch "stable", but otherwise I think it's very similar. We've also discussed supporting hotfixes.

I guess this approach has fallen slightly out of favour in some circles but to be honest I always good experiences using it. So I thought I'd throw it out there as an easy to follow, already established process for handling merges, active development, hot fixes, and releases in the app.
